apps/dashboard/app/apps/user_configuration.rb

```diff
@@ -1,28 +1,58 @@
 class UserConfiguration
 
+  USER_PROPERTIES = [
+    ConfigurationProperty.property(name: :dashboard_header_img_logo, read_from_env: true),
+    # Whether we display the Dashboard logo image
+    ConfigurationProperty.with_boolean_mapper(name: :disable_dashboard_logo, default_value: false, read_from_env: true),
+    # URL to the Dashboard logo image
+    ConfigurationProperty.property(name: :dashboard_logo, read_from_env: true),
+    # Dashboard logo height used to set the height style attribute
+    ConfigurationProperty.property(name: :dashboard_logo_height, read_from_env: true),
+
+    # The dashboard's landing page layout. Defaults to nil.
+    ConfigurationProperty.property(name: :dashboard_layout),
+    # The configured pinned apps
+    ConfigurationProperty.property(name: :pinned_apps, default_value: []),
+    # The length of the "Pinned Apps" navbar menu
+    ConfigurationProperty.property(name: :pinned_apps_menu_length, default_value: 6),
+
+    ConfigurationProperty.property(name: :profile_links, default_value: []),
+  ].freeze
+
   def initialize
     @config = ::Configuration.config
+    add_property_methods
   end
 
-  # The dashboard's landing page layout. Defaults to nil.
-  def dashboard_layout
-    fetch(:dashboard_layout, nil)
+  def brand_bg_color
+    color = ENV.values_at('OOD_BRAND_BG_COLOR', 'BOOTSTRAP_NAVBAR_DEFAULT_BG', 'BOOTSTRAP_NAVBAR_INVERSE_BG').compact.first
+    color || fetch(:brand_bg_color)
   end
 
-  # The configured pinned apps
-  def pinned_apps
-    fetch(:pinned_apps, [])
+  def brand_link_active_bg_color
+    color = ENV.values_at('OOD_BRAND_LINK_ACTIVE_BG_COLOR', 'BOOTSTRAP_NAVBAR_DEFAULT_LINK_ACTIVE_BG','BOOTSTRAP_NAVBAR_INVERSE_LINK_ACTIVE_BG' ).compact.first
+    color || fetch(:brand_link_active_bg_color)
   end
 
-  # The length of the "Pinned Apps" navbar menu
-  def pinned_apps_menu_length
-    fetch(:pinned_apps_menu_length, 6)
+  # Sets the Bootstrap 4 navbar type
+  # See more about Bootstrap color schemes: https://getbootstrap.com/docs/4.6/components/navbar/#color-schemes
+  # Supported values: ['dark', 'inverse', 'light', 'default']
+  # @return [String, 'dark'] Default to dark
+  def navbar_type
+    type = ENV["OOD_NAVBAR_TYPE"] || fetch(:navbar_type)
+    if type == 'inverse' || type == 'dark'
+      'dark'
+    elsif type == 'default' || type == 'light'
+      'light'
+    else
+      'dark'
+    end
   end
 
   # What to group pinned apps by
   # @return [String, ""] Defaults to ""
   def pinned_apps_group_by
-    group_by = fetch(:pinned_apps_group_by, "")
+    group_by = type = ENV["OOD_PINNED_APPS_GROUP_BY"] || fetch(:pinned_apps_group_by, "")
```

Comment:
is this an extra assignment to `type`?

Reply:
No, this is an error on my part. Will remove. It should be:
`group_by = ENV["OOD_PINNED_APPS_GROUP_BY"] || fetch(:pinned_apps_group_by, "")`